module/support/logging/FileLogHandler/src/FileLogHandler.cpp

```diff
@@ -107,16 +107,16 @@ namespace module::support::logging {
 
```

Comment:
Line 100 and 101 there is still some camel case but I cannot review it properly because it wasn't changed in this PR